src/near-rpc-provider.ts

```diff
@@ -184,7 +185,14 @@ export class NearRpcProvider extends JsonRpcProvider {
       case 'getBalance':
         return this._internalGetBalance(params)
       case 'getBlock':
-        return this._internalGetBlock(params)
+        return logger.throwError(
+          'getBlock function is not supported in Near Provider. Please use getBlockWithChunck function',
+          Logger.errors.SERVER_ERROR,
+          {
+            method: 'getBlock',
+            params: params,
+          },
+        )
```

Comment:
I would add a `getBlock` method in the class directly and move this there, so that it throws there directly

Reply:
On it!